_signal_processing/krakenSDR_signal_processor.py

```diff
@@ -239,6 +241,46 @@ def resetPeakHold(self):
         if self.spectrum_fig_type == "Single":
             self.peak_hold_spectrum = np.ones(self.spectrum_window_size) * -200
 
+    def calculate_squelch(self, sampling_freq, N):
+        def is_enabled_auto_squelch(v):
+            return v == "Auto" or (v == "Default" and self.vfo_default_squelch_mode == "Auto")
+
+        def find_nearest(array, value):
+            array = np.asarray(array)
+            idx = (np.abs(array - value)).argmin()
+            return idx, array[idx]
+
+        if self.spectrum_fig_type == "Single":
```

Comment:
@redradist I don't understand why this is needed?

Reply:
Change on CH0